src/commands/dev/dev.mjs

```diff
@@ -710,6 +807,7 @@ export const createDevCommand = (program) => {
     .option('-o ,--offline', 'disables any features that require network access')
     .option('-l, --live', 'start a public live session', false)
     .option('--functionsPort <port>', 'port of functions server', (value) => Number.parseInt(value))
+    .option('-s, --serve', 'run in "serve" mode', false)
```

Comment:
This needs a better description.

Reply:
How about `--prod` instead? We're running it in a prod-like mode

Reply:
Yeah, I like that!

Reply:
@ticachica any thoughts/concerns here?

Reply:
Renamed to `--prod` in https://github.com/netlify/cli/pull/5338/commits/78c35b68bf54712bc6230c0c7d874f5a9ac5c780.